Sources/Apodini/Configurations/HTTPConfiguration.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  HTTPConfiguration.swift
+//  
+//
+//  Created by Tim Gymnich on 18.1.21.
+//
+
+import Foundation
+import NIO
+@_implementationOnly import ConsoleKit
+
+/// A `Configuration` for HTTP.
+/// The configuration can be done in two ways, either via the
+/// command line arguments --hostname, --port and --bind or via the
+/// function `address`
+public class HTTPConfiguration: Configuration {
+    private var address: BindAddress?
+
+    enum HTTPConfigurationError: Error {
+        case incompatibleFlags
```

Comment:
Please add an extension so that throwing this error results in a nice error message.

Reply:
what do you mean with create an extension. Is there a specific protocol I should implement?

Reply:
I think the best one would be LocalizedError.

Reply:
done